examples/plot_interpolate_bad_channels.py

```diff
@@ -51,7 +49,7 @@
 evoked_before.plot(exclude=[])
 
 # compute interpolation (also works with Raw and Epochs objects)
-evoked_after = evoked_before.copy().interpolate_bads_eeg()
+evoked_after = evoked_before.interpolate_bads()
 
-# plot interpolated (prevsious bads)
+# plot interpolated (previous bads)
 evoked_after.plot(exclude=[])
```

Comment:
I agree with @Eric89GXL that `exclude` doesn't make sense in this context. `mark` is also a bit counter-intuitive to me since we are actually unmarking the bad channels, not marking them. `unmark` is fine but not explicit.